genai-perf/README.md

```diff
@@ -633,9 +633,35 @@ export file is `profile_export.json`, the genai-perf file will be exported to
 
 ### Session Options
 
+##### `--num-sessions`
+
+The number of sessions to simulate. This is used when generating synthetic
+session data. (default: `0`)
+
 ##### `--session-concurrency <int>`
 
-The number of concurrenct sessions to benchmark. (default: `None`)
+The number of concurrent sessions to benchmark. This must be specified
```

Comment:
Why not reuse --concurrency arg instead of adding new --session-concurrency?

Reply:
This is how Perf Analyzer does it, so we need to maintain parity across. I think this is a good point, but right now, without this arg, PA (and GAP) would have no way of the user is benchmarking in sessions mode unless we add another argument (e.g. -`--session-mode`).

Tagging @matthewkotila, who may be able to speak to the rationale on the PA side and whether this is a good future direction.

Reply:
num_sessions >= 1 or num_session_turns >=2 would also imply session / multi-turn mode right?

Reply:
Yes. But there would be no way to know that benchmarking mode is session-based for the new payload retriever (`--input-file payload:<file_path>`). It can be used for fixed schedule or multi-turn. It allows the user to provide data for specific payloads.

Reply:
GAP needs to pass PA --fixed-schedule for fixed schedule benchmarking and --session-concurrency for session benchmarking.